package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "clarity-js",
-  "version": "0.2.4",
+  "version": "0.2.5",
```

Comment:
Out of date.

Reply:
Updated

---
In reply to: [250282609](https://github.com/Microsoft/clarity-js/pull/113#discussion_r250282609) [](ancestors = 250282609)